draft-ietf-tls-rfc8447bis.md

```diff
@@ -535,6 +535,19 @@ IANA is requested to add a "Comment" column to the following registries:
 This list of registries is all registries that do not already have a
 "Comment" or "Notes" column or that were not orphaned by TLS 1.3.
 
+# Expert Review of Current and Potential IETF and IRTF Documents
+
+The intent of the Specification Required standard for TLS code points
```

Comment:
@kaduk suggestion to address the above issue.

```suggestion

This section expands upon instructions provided to the designated experts in {{Section 17 of RFC84477}}.

The intent of the Specification Required standard for TLS code points
```

Reply:
It seems like this still requires the DE to (either already "just know" or) look at the registry, follow the link to RFC 8447, and then follow the link to this document as indicated in an "Updates:" header.  It seems like it would be appropriate to update the affected registries to list this document as an additional reference, so that it's directly indicated that this document has bearing on the operations of the registry in question.

Reply:
Okay I think we're good in all but one registry:

In each registry that this document uses a value of "D" to, there is an IANA instructions to:

     Add a reference to this document under the reference heading.

The only registry that this document does not mention that also has a Recommended column is the TLS SignatureScheme registry; that registry was created by RFC 8446; PskKeyExchangeMode also includes a Recommended column and was created by RFC 8446, but this document includes an IANA instruction to update the references to this document. Newly submitted PR #52 instructs IANA to update the reference in the TLS SignatureScheme registry to also include this document.

Reply:
Thank you for going through the registries to see which would be affected, and submitting the (thankfully small) delta!

Reply:
NP

@jsalowey can you double check to make sure I got that right?